iam/api-client/src/test/java/iam/snippets/ServiceAccountTests.java

```diff
@@ -144,7 +147,7 @@ public void stage5_testEnableServiceAccount() {
   }
 
   @Test
-  public void stage6_testServiceAccountDelete() {
+  public void stage6_testServiceAccountDelete() throws IOException {
     DeleteServiceAccount.deleteServiceAccount(PROJECT_ID, SERVICE_ACCOUNT);
     String got = bout.toString();
     assertThat(got, containsString("Deleted service account:"));
```

Comment:
Consider retrieving the service account to confirm that the account has been deleted.